### PR TITLE
Add contributors guideline

### PR DESCRIPTION
**Pull request by arthurclares**
This PR introduces the contributors guideline for the FinOps Toolkit for community members accessing the Toolkit from the webpage rather than the GitHub repository. The goal is to provide a simplified version of how individuals can contribute to the project, encouraging newcomers to get started easily. The guideline invites community members to participate, even if it will require further support, such as guidance on GitHub branches are organized.

### CODE REVIEW

File: docs/_resources/contributors.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  - [Code review](#4-code-review)
+  - [Answering questions](#5-answering-questions)
+  - [Documentation update](#6-documentation-update)
+
+</details>
+
+---
+
+## 📝 FinOps Toolkit Contribution Guide
+
```
Comment (flanakin) [inline docs/_resources/contributors.md:31]: The first descriptive paragraph doesn't usually have a header.
```suggestion
```

File: docs/_resources/contributors.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+parent: Resources
+title: How to Contribute
```
Comment (flanakin) [inline docs/_resources/contributors.md:4]: Sentence casing
```suggestion
title: How to contribute
```

File: docs/_resources/contributors.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  - [Answering questions](#5-answering-questions)
+  - [Documentation update](#6-documentation-update)
+
+</details>
+
+---
+
+## 📝 FinOps Toolkit Contribution Guide
+
+Welcome to the FinOps Toolkit project! As an open-source initiative, our success hinges on the collective effort of the community. Your contributions play a vital role in enhancing and expanding the toolkit's capabilities, making it more valuable and effective.
```
Comment (flanakin) [inline docs/_resources/contributors.md:32]:
```suggestion
Welcome to the FinOps toolkit! As an open-source initiative, our success hinges on the collective effort of the community. Your contributions play a vital role in enhancing and expanding the toolkit's capabilities, making it more valuable and effective.
```

File: docs/_resources/contributors.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+There are three primary ways you can contribute to the success of the project:
```
Comment (flanakin) [inline docs/_resources/contributors.md:34]:
```suggestion
There are several ways you can contribute to the FinOps toolkit:
```

File: docs/_resources/contributors.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+1. 🐛 **Working on existing bugs:** You can assist by identifying and addressing existing bugs listed in our repository. 
+2. 🛠️ **Suggesting and owning a new feature, fixing an existing bug, or suggesting a change for an existing tool:** This involves proposing new features, fixing bugs, or suggesting changes for existing tools.
```
Comment (flanakin) [inline docs/_resources/contributors.md:37]: This is covering creating issues and making code changes. Should we split them for people who are may not be able to contribute code?

File: docs/_resources/contributors.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+
+Welcome to the FinOps Toolkit project! As an open-source initiative, our success hinges on the collective effort of the community. Your contributions play a vital role in enhancing and expanding the toolkit's capabilities, making it more valuable and effective.
+
+There are three primary ways you can contribute to the success of the project:
+
+1. 🐛 **Working on existing bugs:** You can assist by identifying and addressing existing bugs listed in our repository. 
+2. 🛠️ **Suggesting and owning a new feature, fixing an existing bug, or suggesting a change for an existing tool:** This involves proposing new features, fixing bugs, or suggesting changes for existing tools. 
+3. 💡 **Submitting a proposal for a new tool:** If you've developed a FinOps-related tool and want it to be part of the FinOps Toolkit, you can submit a proposal for consideration.
+4. 👀 **Code review:** Code review is a critical aspect of maintaining code quality and ensuring the reliability and functionality of the project. By reviewing existing pull requests (PRs) suggested by community members, you can contribute significantly to the project's success without necessarily creating new code.
```
Comment (flanakin) [inline docs/_resources/contributors.md:39]:
```suggestion
4. 👀 **Reviewing pull requests:** Code reviews are a critical aspect of maintaining code quality and ensuring the reliability and functionality of the project. By reviewing existing pull requests (PRs) suggested by community members, you can contribute significantly to the project's success without necessarily creating new code.
```

File: docs/_resources/contributors.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+5. 💬 **Answering questions:** Being actively involved in answering questions and providing assistance to community members is an invaluable contribution to the project.
+6. 📚 **Documentation update:** Documentation serves as a crucial resource for users, developers, and contributors alike. Keeping documentation accurate, comprehensive, and up-to-date is essential for the success of the toolkit.
```
Comment (flanakin) [inline docs/_resources/contributors.md:41]:
```suggestion
6. 📚 **Updating documentation:** Documentation serves as a crucial resource for users, developers, and contributors alike. Keeping documentation accurate, comprehensive, and up to date is essential for the success of the toolkit.

```

File: docs/_resources/contributors.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+
+## 📝 FinOps Toolkit Contribution Guide
+
+Welcome to the FinOps Toolkit project! As an open-source initiative, our success hinges on the collective effort of the community. Your contributions play a vital role in enhancing and expanding the toolkit's capabilities, making it more valuable and effective.
+
+There are three primary ways you can contribute to the success of the project:
+
+1. 🐛 **Working on existing bugs:** You can assist by identifying and addressing existing bugs listed in our repository. 
+2. 🛠️ **Suggesting and owning a new feature, fixing an existing bug, or suggesting a change for an existing tool:** This involves proposing new features, fixing bugs, or suggesting changes for existing tools. 
+3. 💡 **Submitting a proposal for a new tool:** If you've developed a FinOps-related tool and want it to be part of the FinOps Toolkit, you can submit a proposal for consideration.
```
Comment (flanakin) [inline docs/_resources/contributors.md:38]:
```suggestion
3. 💡 **Proposing a new tool:** If you've developed a FinOps-related tool and want it to be part of the FinOps toolkit, you can submit a proposal for consideration.
```

File: docs/_resources/contributors.md [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+4. 👀 **Code review:** Code review is a critical aspect of maintaining code quality and ensuring the reliability and functionality of the project. By reviewing existing pull requests (PRs) suggested by community members, you can contribute significantly to the project's success without necessarily creating new code.
+5. 💬 **Answering questions:** Being actively involved in answering questions and providing assistance to community members is an invaluable contribution to the project.
+6. 📚 **Documentation update:** Documentation serves as a crucial resource for users, developers, and contributors alike. Keeping documentation accurate, comprehensive, and up-to-date is essential for the success of the toolkit.
+
+We appreciate your interest in contributing to the FinOps Toolkit project!
+
+### 🐛 1. Working on existing bugs
```
Comment (flanakin) [inline docs/_resources/contributors.md:45]:
```suggestion
<br>

## 🐛 Working on existing bugs
```

File: docs/_resources/contributors.md [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+   <summary class="fs-2 text-uppercase">On this page</summary>
+
+
+- [📝 How to Contribute](#-finops-toolkit-contribution-guide)
+  - [Working on existing bugs](#1-working-on-existing-bugs)
+  - [Suggesting and owning a new feature, fixing a bug, or suggesting a change for an existing Tool](#2-suggesting-and-owning-a-new-feature-fixing-a-bug-or-suggesting-a-change-for-an-existing-tool)
+  - [Submitting a proposal for a new tool](#3-submitting-a-proposal-for-a-new-tool)
+  - [Code review](#4-code-review)
+  - [Answering questions](#5-answering-questions)
+  - [Documentation update](#6-documentation-update)
```
Comment (flanakin) [inline docs/_resources/contributors.md:24]:
```suggestion
- [Working on existing bugs](#-working-on-existing-bugs)
- [Suggesting and owning a new feature, fixing a bug, or suggesting a change for an existing Tool](#-suggesting-and-owning-a-new-feature-fixing-a-bug-or-suggesting-a-change-for-an-existing-tool)
- [Proposing a new tool](#-proposing-a-new-tool)
- [Reviewing pull requests](#-reviewing-pull-requests)
- [Answering questions](#-answering-questions)
- [Updating documentation](#-updating-documentation)
```

File: docs/_resources/contributors.md [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+5. 💬 **Answering questions:** Being actively involved in answering questions and providing assistance to community members is an invaluable contribution to the project.
+6. 📚 **Documentation update:** Documentation serves as a crucial resource for users, developers, and contributors alike. Keeping documentation accurate, comprehensive, and up-to-date is essential for the success of the toolkit.
+
+We appreciate your interest in contributing to the FinOps Toolkit project!
+
+### 🐛 1. Working on existing bugs
+
+If you're eager to contribute but unsure where to begin, a great starting point is to review the list of existing feedback and identify issues that align with your skills and interests. 
+
+1. Look through the list of existing bugs in the [Issues tab](https://github.com/finopsfoundation/FinOpsToolkit/issues).
```
Comment (flanakin) [inline docs/_resources/contributors.md:49]:
```suggestion
1. Look through the list of [existing issues](https://github.com/microsoft/finops-toolkit/issues).
```

File: docs/_resources/contributors.md [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+2. Assign yourself to the bug you want to work on.
+
+If this is the first time you are working on the FinOps Toolkit repository, make sure to review our [Get Started Wiki.](https://github.com/microsoft/finops-toolkit/wiki#-test-and-verify)
```
Comment (flanakin) [inline docs/_resources/contributors.md:52]:
```suggestion
If this is the first time you are working on the FinOps toolkit repository, make sure to review our [Get started wiki](https://github.com/microsoft/finops-toolkit/wiki#-get-started).

```

File: docs/_resources/contributors.md [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+
+Once you are assigned to the bug, GitHub will automatically assign the tag "Community Contributor" to inform the team responsible for that tool of your contribution.
```
Comment (flanakin) [inline docs/_resources/contributors.md:54]: Is this something you're proposing? It doesn't do this today.

File: docs/_resources/contributors.md [hunk 14]
```diff
@@ -0,0 +1,124 @@
...
+If you're eager to contribute but unsure where to begin, a great starting point is to review the list of existing feedback and identify issues that align with your skills and interests. 
+
+1. Look through the list of existing bugs in the [Issues tab](https://github.com/finopsfoundation/FinOpsToolkit/issues).
+2. Assign yourself to the bug you want to work on.
+
+If this is the first time you are working on the FinOps Toolkit repository, make sure to review our [Get Started Wiki.](https://github.com/microsoft/finops-toolkit/wiki#-test-and-verify)
+
+Once you are assigned to the bug, GitHub will automatically assign the tag "Community Contributor" to inform the team responsible for that tool of your contribution. 
+
+### 🛠️ 2. Suggesting and owning a new feature, fixing an existing bug, or suggesting a change for an existing tool
```
Comment (flanakin) [inline docs/_resources/contributors.md:56]:
```suggestion
<br>

## 🛠️ Suggesting and owning a new feature, fixing an existing bug, or suggesting a change for an existing tool
```

File: docs/_resources/contributors.md [hunk 15]
```diff
@@ -0,0 +1,124 @@
...
+
+#### Submitting a New Issue and Willing to Fix
```
Comment (flanakin) [inline docs/_resources/contributors.md:58]:
```suggestion
### Submitting a new issue
```

File: docs/_resources/contributors.md [hunk 16]
```diff
@@ -0,0 +1,124 @@
...
+
+If you have a new feature suggestion, identified a bug, or have ideas for improving existing tools, we encourage you to submit an issue. Your feedback helps us ensure that the toolkit meets the needs of its users. And if you're willing to take it a step further and contribute to implementing your suggestions, we welcome your contributions with open arms! 
+
+To do that, follow these steps: 
+
+1. Go to the [Issues tab](https://github.com/finopsfoundation/FinOpsToolkit/issues) and click on "New issue".
```
Comment (flanakin) [inline docs/_resources/contributors.md:64]:
```suggestion
1. Go to the [issues list](https://github.com/finopsfoundation/FinOpsToolkit/issues) and click on "New issue".
```

File: docs/_resources/contributors.md [hunk 17]
```diff
@@ -0,0 +1,124 @@
...
+2. Choose the option that best describes your contribution (bug fix, change request, or feature request).
+3. Check the box "Willing to fix" at the bottom of the page.
```
Comment (flanakin) [inline docs/_resources/contributors.md:66]: If they're willing to fix it, should we just ask them to submit a PR directly? I'm thinking we just remove this whole "willing to fix" part.

File: docs/_resources/contributors.md [hunk 18]
```diff
@@ -0,0 +1,124 @@
...
+4. Submit your issue.
+
+Your issue will be reviewed by the FinOps Toolkit governance team, who will reach out to you to discuss further and give you the green light to start developing. 
+
+### 3. 💡 Submitting a proposal for a new tool
```
Comment (flanakin) [inline docs/_resources/contributors.md:71]:
```suggestion
<br>

## 💡 Proposing a new tool
```

File: docs/_resources/contributors.md [hunk 19]
```diff
@@ -0,0 +1,124 @@
...
+
+If you have developed a FinOps-related tool and want it to be part of the FinOps Toolkit, follow these steps:
+
+1. Send an email to [ftk-leads@microsoft.com](mailto:ftk-leads@microsoft.com) with your proposal.
+2. Include as much information as possible about your tool and how it can benefit the FinOps community.
+
+The FinOps toolkit governance committee will review your proposal and reach out to you for further discussion.
+
```
Comment (flanakin) [inline docs/_resources/contributors.md:79]:
```suggestion
If you have developed a FinOps-related tool and want it to be part of the FinOps toolkit, send the details to [ftk-leads@microsoft.com](mailto:ftk-leads@microsoft.com?subject=[FTK]+New+tool+for+the+FinOps+toolkit). Please include as much information as possible about your tool and how it can benefit the FinOps community.

The FinOps toolkit governing board will review your proposal and reach out to you for further discussion.
```

File: docs/_resources/contributors.md [hunk 20]
```diff
@@ -0,0 +1,124 @@
...
+1. Go to the [Issues tab](https://github.com/finopsfoundation/FinOpsToolkit/issues) and click on "New issue".
+2. Choose the option that best describes your contribution (bug fix, change request, or feature request).
+3. Check the box "Willing to fix" at the bottom of the page.
+4. Submit your issue.
+
+Your issue will be reviewed by the FinOps Toolkit governance team, who will reach out to you to discuss further and give you the green light to start developing. 
+
+### 3. 💡 Submitting a proposal for a new tool
+
+If you have developed a FinOps-related tool and want it to be part of the FinOps Toolkit, follow these steps:
```
Comment (flanakin) [inline docs/_resources/contributors.md:73]: Any reason we don't want them to just create an issue?

File: docs/_resources/contributors.md [hunk 21]
```diff
@@ -0,0 +1,124 @@
...
+
+1. Send an email to [ftk-leads@microsoft.com](mailto:ftk-leads@microsoft.com) with your proposal.
+2. Include as much information as possible about your tool and how it can benefit the FinOps community.
+
+The FinOps toolkit governance committee will review your proposal and reach out to you for further discussion.
+
+
+### 4. 👀 **Code review**
```
Comment (flanakin) [inline docs/_resources/contributors.md:81]:
```suggestion
<br>

## 👀 **Reviewing pull requests**
```

File: docs/_resources/contributors.md [hunk 22]
```diff
@@ -0,0 +1,124 @@
...
+
+- **Review Pull Requests:** Regularly check the list of open PRs on the repository and provide constructive feedback on code changes proposed by community members. Ensure that the code adheres to our [coding guidelines](https://github.com/microsoft/finops-toolkit/wiki/Coding-guidelines.md), is well-documented, and follows best practices.
```
Comment (flanakin) [inline docs/_resources/contributors.md:83]:
```suggestion
Check the list of [open pull requests](https://github.com/microsoft/finops-toolkit/pulls) and provide constructive feedback on code changes proposed by community members. Ensure that the code adheres to our [coding guidelines](https://github.com/microsoft/finops-toolkit/wiki/Coding-guidelines), is well-documented, and follows best practices.
```

File: docs/_resources/contributors.md [hunk 23]
```diff
@@ -0,0 +1,124 @@
...
+
+When possible, try to suggest code changes using **Add a suggestion** to streamline the review process.
+
+![Screenshot of the 'Add a suggestion' command in pull request reviews](https://user-images.githubusercontent.com/399533/179936119-88c10c44-f181-4fa3-83b8-91376c8e4c58.png)
+
+<br>
+
+- **Improve Quality:** Offer suggestions for improving the quality of the code, such as optimizing performance, enhancing readability, and addressing potential edge cases or bugs.
+
+- **Validate Functionality:** Test the functionality of the code changes proposed in the PR to ensure they work as intended. This involves running tests, verifying edge cases, and checking for any unintended side effects.
```
Comment (flanakin) [inline docs/_resources/contributors.md:93]:
```suggestion
Offer suggestions for improving the quality of the code, such as optimizing performance, enhancing readability, and addressing potential edge cases or bugs.

Test the functionality of the code changes proposed in the PR to ensure they work as intended. This involves running tests, verifying edge cases, and checking for any unintended side effects.

When possible, try to suggest code changes using **Add a suggestion** to streamline the review process.

![Screenshot of the 'Add a suggestion' command in pull request reviews](https://user-images.githubusercontent.com/399533/179936119-88c10c44-f181-4fa3-83b8-91376c8e4c58.png)

<br>
```

File: docs/_resources/contributors.md [hunk 24]
```diff
@@ -0,0 +1,124 @@
...
+
+### 5. 💬 Answering Questions
```
Comment (flanakin) [inline docs/_resources/contributors.md:95]:
```suggestion
## 💬 Answering questions
```

File: docs/_resources/contributors.md [hunk 25]
```diff
@@ -0,0 +1,124 @@
...
+
+Being actively involved in answering questions and providing assistance to community members is an invaluable contribution to the project.
+
+- **Share Knowledge:** Utilize your expertise and experience to provide helpful and informative responses to questions raised by community members. Offer clear explanations, examples, and references to support your answers.
+
+- **Help Resolve Issues:** Assist community members in troubleshooting technical issues, or resolving configuration problems they encounter while using the toolkit.
+
+### 6. 📚 Documentation Update:
```
Comment (flanakin) [inline docs/_resources/contributors.md:103]:
```suggestion
<br>

## 📚 Updating documentation
```

File: docs/_resources/contributors.md [hunk 26]
```diff
@@ -0,0 +1,124 @@
...
+
+- **Validate Functionality:** Test the functionality of the code changes proposed in the PR to ensure they work as intended. This involves running tests, verifying edge cases, and checking for any unintended side effects.
+
+### 5. 💬 Answering Questions
+
+Being actively involved in answering questions and providing assistance to community members is an invaluable contribution to the project.
+
+- **Share Knowledge:** Utilize your expertise and experience to provide helpful and informative responses to questions raised by community members. Offer clear explanations, examples, and references to support your answers.
+
+- **Help Resolve Issues:** Assist community members in troubleshooting technical issues, or resolving configuration problems they encounter while using the toolkit.
```
Comment (flanakin) [inline docs/_resources/contributors.md:101]:
```suggestion
- **Share knowledge:** Utilize your expertise and experience to provide helpful and informative responses to questions raised by community members. Offer clear explanations, examples, and references to support your answers.
- **Help resolve issues:** Assist community members in troubleshooting technical issues, or resolving configuration problems they encounter while using the toolkit.
```

File: docs/_resources/contributors.md [hunk 27]
```diff
@@ -0,0 +1,124 @@
...
+
+### 6. 📚 Documentation Update:
+
+- **Identify Areas for Improvement:** Review the existing documentation to identify any outdated information, inaccuracies, or gaps that need to be addressed.
+
+- **Improve Clarity and Consistency:** Clarify ambiguous sections, reorganize content for better structure, and ensure consistency in terminology, formatting, and style throughout the documentation.
+
+- **Update Content:** Update outdated information, add new content to cover recent features or changes, and remove obsolete sections that are no longer relevant.
```
Comment (flanakin) [inline docs/_resources/contributors.md:109]:
```suggestion
- **Identify areas for improvement:** Review the existing documentation to identify any outdated information, inaccuracies, or gaps that need to be addressed.
- **Improve clarity and consistency:** Clarify ambiguous sections, reorganize content for better structure, and ensure consistency in terminology, formatting, and style throughout the documentation.
- **Update content:** Update outdated information, add new content to cover recent features or changes, and remove obsolete sections that are no longer relevant.
```

File: docs/_resources/contributors.md (hunk 13 above)
Comment (arthurclares) [reply docs/_resources/contributors.md:56]: Yes, I was hoping we could add a GitHub policy to assign this tag. I'm thinking about tracking the work done by the community.
 I will remove it so we can merge this PR but add to our backlog

File: docs/_resources/contributors.md (hunk 17 above)
Comment (arthurclares) [reply docs/_resources/contributors.md:67]: Maybe they are willing to fix but they are not so comfortable working with GitHub (branches can be scary 👻) or they want to fix it, but are unsure how to proceed, etc. The idea of this checkbox is just a way for us to know that there is a person trying to fix that, and we may want to reach out to them and see if they need any help with it. 
Also, maybe there is value on reporting how many people are trying to fix these issues.

**Comment by arthurclares** [comment]
@flanakin Only thing left is the "willing to fix" piece. I'm happy either way.
If you are happy to keep it, just resolve the conversation and this PR will be merged to the Level Up branch.

**Comment by arthurclares** [comment]
@flanakin Only thing left is the "willing to fix" piece. I'm happy either way.
If you are happy to keep it, just resolve the conversation and this PR will be merged to the Level Up branch.

File: .github/ISSUE_TEMPLATE/propose-tool.md [hunk 28]
```diff
@@ -0,0 +1,31 @@
+---
+name: "\U+1F4CB Propose new tool"
```
Comment (flanakin) [inline .github/ISSUE_TEMPLATE/propose-tool.md:2]:
```suggestion
name: "Propose new tool"

```

**Review by flanakin — APPROVED** [review]
